src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -250,6 +250,38 @@ public async Task InspectLocalsInPreviousFramesDuringSteppingIn(bool use_cfo)
             );
         }
 
+        [Fact]
+        public async Task InspectNullStringBuilderLocalsDuringSteppingIn()
+        {
+            // testing uninitialized MONO_TYPE_CLASS reference
+            Console.WriteLine(await SetBreakpoint("dotnet://debugger-test.dll/debugger-test.cs", 562, 8));
+            await EvaluateAndCheck("window.setTimeout(function() { invoke_outer_method(); }, 1);", "dotnet://debugger-test.dll/debugger-test.cs", 562, 8, "OuterMethod");
+
+            await StepAndCheck(StepKind.Into, "dotnet://debugger-test.dll/debugger-test.cs", 566, 4, "InnerMethod",
+                locals_fn: (locals) =>
+                {
+                    Assert.Equal(1, locals.Count());
+                    CheckObject(locals, "sb", "System.Text.StringBuilder");
+                }
+            );
+        }
+
+        [Fact]
+        public async Task InspectNullBoxedLocalsDuringSteppingIn()
```

Comment:
Do we know why this isn't getting caught in `InspectBoxedLocals`, which has a `o_null` case in  https://github.com/pavelsavara/runtime/blob/pr_wasm_debugger_crash/src/mono/wasm/debugger/tests/debugger-test/debugger-test.cs#L358 ?

Reply:
Ugh.. my fault

Reply:
@radical Could you please create separate issue about it? 
I'm not comfortable mixing it with this change. 
Especially because I don't know what is the "correct" shape of the json for null object.